src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/TaskManagementGetTaskProviderTest.java

```diff
@@ -203,8 +203,12 @@ public Task createWaTask() {
         final TaskPermissions permissions = new TaskPermissions(
             Set.of(
                 PermissionTypes.READ,
-                PermissionTypes.EXECUTE,
-                PermissionTypes.REFER
+                PermissionTypes.COMPLETE,
+                PermissionTypes.CLAIM,
+                PermissionTypes.UNCLAIM,
+                PermissionTypes.UNASSIGN_CLAIM,
+                PermissionTypes.CANCEL,
+                PermissionTypes.EXECUTE
```

Comment:
Should this include UnassignAssign & UnclaimAssign?
Should Cancel be on this particular list?

Reply:
From base branch 1528